Firestore/core/src/local/persistence.h

```diff
@@ -133,6 +142,10 @@ class Persistence {
    */
   virtual ReferenceDelegate* reference_delegate() = 0;
 
+  /** Releases components created for users other than given `target_uid`. */
```

Comment:
Please update this documentation to also specify under which conditions it is appropriate to invoke this new `ReleaseOtherUserSpecificComponents()` method.

Reply:
Done.